lib/csrf.js

```diff
@@ -67,16 +67,12 @@ class CSRF {
     createToken();
 
     if (verify.error) {
-      try {
-        return ctx.error(verify);
-      } catch (e) {
-        console.log(e);
-      }
+      throw new Boom(verify.error.message);
```

Comment:
I confirmed that this works if you just change lines 70-74 to this:
```
if (verify.error) {
      return ctx.error(verify);
    }
```
The 'catch' was making the error we throw fail silently